library/std/src/sys/windows/path.rs

```diff
@@ -1,13 +1,34 @@
+use super::{c, fill_utf16_buf, to_u16s};
 use crate::ffi::OsStr;
+use crate::io;
 use crate::mem;
+use crate::path::Path;
 use crate::path::Prefix;
+use crate::ptr;
 
 #[cfg(test)]
 mod tests;
 
 pub const MAIN_SEP_STR: &str = "\\";
 pub const MAIN_SEP: char = '\\';
 
+// UTF-16 encoded code points, used in parsing and building UTF-16 paths.
+// All of these are in the ASCII range so they can be cast directly to `u16`.
+const SLASH: u16 = b'\\' as _;
```

Comment:
Nit: Maybe rename `SLASH` to `BACKSLASH`and move those constants into `to_verbatim()` as they are not used anywhere else?

Reply:
Hm, how about something like `SEP`? I'm keen to keep it short because I think it's easier to read when used repeatedly.

Reply:
`SEP` would be fine.